CMakeLists.txt

```diff
@@ -21,8 +21,8 @@ if (CMAKE_C_COMPILER_ID STREQUAL Clang)
     set(CMAKE_COMPILER_IS_GNUCXX OFF)
 endif()
 
-# set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -pthread -Wall -Wextra -std=gnu99 -D__USE_XOPEN")
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -std=gnu99 -D__USE_XOPEN")
+# set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -pthread -Wall -Wextra -std=gnu99 -D__USE_XOPEN -fPIC")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -std=gnu99 -D__USE_XOPEN -fPIC")
```

Comment:
> Fixes the compilation without -fPIC flag, which caused error during
> preparing packages for Ubuntu Xenial. This flag was added
> in set(CMAKE_C_FLAGS ...) statement in CMakeLists.txt.

Can you reveal more info about the problem? I tried to build a package without this change for Ubuntu Xenial and it succeeds. What also confuses me: AFAIR, cmake enables `-fPIC` for shared libraries by default.

And I would also link [this](https://stackoverflow.com/q/38296756/1598057) StackOverflow question about most idiomatic way to enforcing position independent code (if we really need to set it explicitly).

Reply:
AFAIR, I tried packaging for Xenial without -fPIC flag and it didn't succeed. I've removed this flag and have tried again – now it passes.

Reply:
Okay so.